src/core/transport/ntcp_session.cc

```diff
@@ -437,8 +437,8 @@ void NTCPSession::SendPhase3() {
     i2p::context.GetIdentity().ToBuffer(
         buf,
         static_cast<std::size_t>(NTCPSize::buffer));
-  std::uint32_t tsA = htobe32(i2p::util::GetSecondsSinceEpoch());
-  htobuf32(buf, tsA);
+  std::uint32_t ts_a = htobe32(i2p::util::GetSecondsSinceEpoch());
```

Comment:
`a` can remain uppercase as we need to differentiate between timestamp A and B. Even better, rename to `timestamp_A`.